src/Resolution.ts

```diff
@@ -265,14 +286,26 @@ export default class Resolution {
    *  - BTC
    *  - ETH
    * @throws [[ResolutionError]] if address is not found
+   * @depricated since v1.6.2 use Resolution.addr instead
```

Comment:
`.` indicates a static method `#` indicates an instance method.

```suggestion
   * @deprecated since v1.6.2 use Resolution#addr instead
```